### PR TITLE
docs: put lock file considerations list into table

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes:

- Put the list items for the lock file considerations in a table
- Fix semantic errors that were present in the list

### Current list for reference:

- The more groups you use, the separate PRs you have, and hence the less overall chance at lock file conflicts
- The less your schedule, the more chance that you pile up concurrent PRs, which increases the chance of lock file conflicts
- The more automerging you do, the less chance you have concurrent PRs, which decreases the chance of lock file conflicts

### Why this list is confusing to me

- `the separate PRs` doesn't say if you get _more_ or _less_ separate PRs.
- `The less your schedule` is vague, I think it should be something like `if you don't schedule enough time for Renovate to run`.
- `the less chance you have concurrrent PRs` should be `the less chance you have of concurrent PRs`, at least I think that's correct.

### What I did to make it better

I scrapped the list, and put the corrected information in a table.
Let me know if you like/hate this. :wink: 

## Why is this a draft PR

Line 130 will need some work after the table idea is approved.

### CODE REVIEW

**Review by rarkins — DISMISSED** [review]

File: docs/usage/noise-reduction.md [hunk 1]
```diff
@@ -129,9 +129,11 @@ Have you come up with a rule that you think others would benefit from? How about
 
 As mentioned earlier, using lock files greatly increases the chance that merging one PR will result in a second PR becoming conflicted with `master`. Therefore:
```
Comment (JamieMagee) [inline docs/usage/noise-reduction.md:130]: You mentioned this PR is still in draft due to this line. Here's a suggestion

```suggestion
Using lock files greatly increases the chance that merging one PR will result in a second PR becoming conflicted with `master`. The table below highlights different noise reduction strategies and their affect on pull request and potential lock file conflicts:
```

File: docs/usage/noise-reduction.md [hunk 2]
```diff
@@ -129,9 +129,11 @@ Have you come up with a rule that you think others would benefit from? How about
...
 
-- The more groups you use, the separate PRs you have, and hence the less overall chance at lock file conflicts
-- The less your schedule, the more chance that you pile up concurrent PRs, which increases the chance of lock file conflicts
-- The more automerging you do, the less chance you have concurrent PRs, which decreases the chance of lock file conflicts
+| Action                               | Effect on pull requests          | Chance of lock file conflicts |
+| ------------------------------------ | -------------------------------- | ----------------------------- |
+| Group dependencies together          | Reduces number of separate PRs   | Decreases                     |
+| Automerge dependencies               | Reduces number of concurrent PRs | Decreases                     |
+| Decrease scheduled time for Renovate | Increases concurrent PRs         | Increases                     |
```
Comment (JamieMagee) [inline docs/usage/noise-reduction.md:136]: Small changes to fit the pattern

```suggestion
| Group dependencies together          | Decreases separate PRs           | Decreases                     |
| Automerge dependencies               | Decreases concurrent PRs         | Decreases                     |
| Decrease scheduled time for Renovate | Increases concurrent PRs         | Increases                     |
```

**Review by JamieMagee — DISMISSED** [review]

**Comment by HonkingGoose** [comment]
Affect is not really the right word in this context, it should be effect.
So I changed the word.

Also the linter was failing, so I re-ran Prettier to format the table.

**Comment by HonkingGoose** [comment]
Also please let me know if this needs to be squashed down into a single clean commit before it can be merged, or if you do your own squash merges?

**Comment by JamieMagee** [comment]
> Affect is not really the right word in this context, it should be effect.
So I changed the word.

Good catch. Thank you!

> Also please let me know if this needs to be squashed down into a single clean commit before it can be merged, or if you do your own squash merges?

No need to squash, we'll squash merge it to `master`

**Comment by HonkingGoose** [comment]
I'll stay away from this branch then, so that you can do the merge yourself. :smile:

**Review by JamieMagee — APPROVED** [review]

**Pull request merged**